codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 0%
+        base: auto
+        flags:
+          - unit
+        paths:
+          - "src"
+       # advanced settings
+        branches:
+          - master
+        if_ci_failed: error #success, failure, error, ignore
+        informational: false
```

Comment:
Should information be `true`?

Reply:
Sorry for the confusing. The old commit is just default values. I've updated it and it should disable both project and patch status check.

Reply:
Thanks 👍 

Two things here for PR.
- Can we still see the coverage in each PR status check? but don't mark the checks failed even if the coverage decreases. (I feel the informational setting of the patch status can help)
- Given the first, don't add a coverage comment in the PR

Reply:
I've modified it and it should disable comment and let PR status check always pass.